sematic/resolvers/tests/test_silent_resolver.py

```diff
@@ -22,3 +24,31 @@ def pipeline(a: float, b: float) -> float:
 
 def test_silent_resolver():
     assert SilentResolver().resolve(pipeline(3, 5)) == 24
+
+
+_tried = 0
+
+
+class SomeException(Exception):
+    pass
+
+
+@func(retry=RetrySettings(exceptions=(SomeException,), times=3))
+def try_three_times():
```

Comment:
```suggestion
def retry_three_times():
```
"try three times" to me says it will execute 3 times, not 4